contract_manager/src/store.ts

```diff
@@ -143,6 +148,20 @@ export class Store {
     });
   }
 
+  loadAllTokens() {
```

Comment:
the logic here is very similar to `loadAllVaults` and somewhat similar to `loadAllChains`. Possible to reduce the duplicate code.

Reply:
I don't think this is worth it -- the abstracted out code is going to be much harder to understand.